### ISSUE TITLE
on_mouse_press() won't be called for a short time after a key press

### ISSUE THREAD

**Issue by SirGnip** [pvcraven/arcade#551]
### Actual behavior:
A mouse click that happens shortly after a key press won't trigger an on_mouse_press() method. 
 Mouse clicks that happen a second (very roughly) or more after a keypress will trigger the on_mouse_press() method.

### Expected behavior:
Every mouse click should trigger an on_mouse_press() method call.

### Steps to reproduce/example code:
```
import arcade

class MyGame(arcade.Window):
    def __init__(self):
        super().__init__(300, 300, 'click after key bug repro')

    def on_mouse_press(self, x: float, y: float, button: int, modifiers: int):
        print('****** click', x, y)

if __name__ == '__main__':
    game = MyGame()
    arcade.run()
```

* Run this code
* Tap any key and then click the mouse button quickly and repeatedly.
* Observe that the initial mouse clicks do not generate "click" messages.
* You can repeat these steps (tap a key and click the mouse) over and over.

I was able to repro this with arcade 2.2.2, 2.1.7, 2,0.9, 2.0.0 and even 1.3.5 on a Windows 10 laptop.  Can anyone else repro this behavior, or is this "just me"?

**Comment by SirGnip** [comment]
Sorry. This is machine specific.  I was able to repro this behavior in other applications on my laptop (ex: Notepad).  And, the behavior only happens when I'm using the laptop keyboard and built-in touchpad.  If I connect a USB mouse, the mouse doesn't exhibit the behavior.  

Apologies that I didn't test this before creating an Issue.

(Apparently, Lenovo ThinkPads have a "delay" feature on their touchpads?  I had to do a registry hack to turn it off.  Sigh...)
